ui/src/screens/Jobs/hooks/useJobsReducer.ts

```diff
@@ -119,8 +115,9 @@ export default function usePostsReducer(): [JobsState, JobsActions] {
   const id = match?.params?.id;
 
   React.useEffect(() => {
-    axios.get<JobsResponseData>(`${process.env.REACT_APP_API}/jobs`).then((res) => {
-      updateJobs(res.data.jobs);
+    axios.get<PostInterface[]>(`${process.env.REACT_APP_API}/jobs/1`).then((res) => {
```

Comment:
In #93, I introduce `JobsGoWhereApiClient` which is axios underneath but allow us to just pass relative URL + its authorisation token is send whenever possible.